content/en/building/concepts/access.md

```diff
@@ -55,7 +55,20 @@ Users may log out by going to the options menu available in the top right corner
 
 {{< figure src="login-mobile.png" link="login-mobile.png" class="left col-3 col-lg-3" >}}
 
-{{< figure src="login-desktop.png" link="login-desktop.png" class="left col-9 col-lg-9" >}}
+{{< figure src="login-desktop.png" link="login-desktop.png" class="right col-8 col-lg-8" >}}
+
+### Password reset on first login
+
+Added in 4.16.0.
```

Comment:
```suggestion
**Added in 4.16.0.**
```

Reply:
Thank you! I've added all the suggestions